samples/My.Hr/My.Hr.Api/appsettings.json

```diff
@@ -4,5 +4,8 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "ConnectionStrings": {
+    "Database": "Data Source=.;Initial Catalog=My.HrDb;Integrated Security=True;TrustServerCertificate=true"

```

Comment:
Hmm. I had that. It broke the GitHub action, as that supersedes the env var.

Reply:
Because I had that (removed from app settings) in my infra PR and tests passed.

I had some other issues with having it in app settings. I think it broke the app after deployment

Reply:
I think I have fixed it now. The connection string has been moved to `development` and the `unittest` now sets env var to enable for unit tests. I need to update `UnitTestEx` to explicitly set env var to `Development` similar to code for functions to minimize need for this hack.